x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/KibanaOwnedReservedRoleDescriptors.java

```diff
@@ -215,6 +215,7 @@ static RoleDescriptor kibanaSystem(String name) {
                         ".logs-endpoint.action.responses-*",
                         ".logs-endpoint.diagnostic.collection-*",
                         ".logs-endpoint.actions-*",
+                        "logs-endpoint.heartbeat-*",
```

Comment:
Can this index follow the same pattern as the others? The preceding `.` will help to avoid confusion with user created data indicies. See [here](https://docs.elastic.dev/kibana-dev-docs/key-concepts/security-intro#the-kibana_system-user)